akka-http-caching/src/main/mima-filters/10.1.8.backwards.excludes

```diff
@@ -0,0 +1,9 @@
+# Scala 2.13.0-M5
```

Comment:
```suggestion
# Type hierarchy of `CachingSettings` changed because `SettingCompanionImpl` is not a
# supertype any more. `SettingCompanionImpl` is `@InternalApi` so it's ok.
```

Reply:
Thanks for your suggestion!